src/com/facebook/buck/util/concurrent/BUCK.autodeps

```diff
@@ -9,6 +9,7 @@
       "//src/com/facebook/buck/util/immutables:immutables",
       "//third-party/java/concurrent-locks:concurrent-locks",
       "//third-party/java/guava:guava",
+      "//third-party/java/errorprone:error-prone-annotations",
```

Comment:
@davido autodeps should understand dependencies on 3rd party JARs, but if it isn't picking up a checked-in jar for some reason you can force an explicit import->dep mapping through .buckconfig. We already do this for caliper:
```
[autodeps]
   java-package-mappings = com.google.caliper => //third-party/java/caliper:caliper
```